index.html

```diff
@@ -0,0 +1,258 @@
+              <div class="uber-frame">
+                <ul class="auto-select">
+                  <li class="client-name">Uber</li>
+                  <li class="counter-uber"></li>
+                  <li class="role-uber">Lead Developer</li>
+                  <li class="counter-tff"></li>
+                </ul>
+                <div class="uber-label">2018</div>
+              </div>
+            </div>
+            <p class="ubertext">A smart assistant to make driving more safe, efficient, and fun by
+              unlocking your most expensive computer: your car.</p>
+            <div class="ubertags">
+              <div class="htmltag">html</div>
+              <div class="rubytag" >Ruby on rails</div>
+              <div class="csstag">css</div>
+              <div class="jscripttag">javaScript</div>
+            </div>
+            <div class="multipost_button">See Project</div>
+          </div>
+        </div>
+        <img class="uberimage" src="images\uberimages.png"/>
+      </section>
+</section>
+
+<!-- About Myself Section -->
+<section class="about-myself" id="About">
+  <div class="content-about-myself">
+    <h2>About Myself</h2>
+    <p>Hello I’m a software developer! I can help you build a product , 
+      feature or website. Look through some of my work and experience! 
+      If you like what you see and have a project you need coded, 
+      don’t hesitate to contact me.</p>
+    <a class="connect">LET'S CONNECT</a>
+    <ul class="social-links">
+      <li><a href="https://twitter.com/obibaadoma">
+        <img src="images\twitter.png" alt="Twitter icon" class="icon"/></a></li>
+      <li><a href="https://www.linkedin.com/in/derek-akrasi-konadu-187453151/">
+        <img src="images\linkedin.png" alt="LinkedIn icon" class="icon"/></a></li>
+      <li><a><img src="images\medium.png" alt="Medium icon" class="icon"/></a></li>
+      <li><a href="https://github.com/obibaadoma">
+        <img src="images\git.png" alt="GitHub icon" class="icon"/></a></li>
+      <li><a><img src="images\angelist.png" alt="Angelist icon" class="icon"/></a></li>
+    </ul>
+    <button type="submit" class="main-buttons">Get my resume</button>
+  </div>
+  <ul class="about-myself-ul">
+    <li class="dropdowns" id="open">
+      <a>Languages</a>
+      <button type="submit"><img src="images\Icon-Chevron-Down-Larger.png" 
+        alt="Dropdown button"></button>
+    </li>
+    <li>
+      <ul class="languages-ul">
+        <li class="languages" id="js">JavaScript</li>
+        <li class="languages" id="html">HTML</li>
+        <li class="languages" id="css">CSS</li>
+      </ul>
+    </li>
+    <li class="dropdowns">
+      <a>Frameworks</a>
+      <button type="submit" ><img src="images\Disabled.png" alt="Dropdown button"></button>
+    </li>
+    <li class="dropdowns">
+      <a>Skills</a>
+      <button type="submit"><img src="images\Disabled.png" alt="Dropdown button"></button>
+    </li>
+  </ul>
+</section>
```

Comment:
- [ ] As per requirements to stick to the `Figma` design as much as possible, kindly refer back to the template to use the same fonts as per the specs.

 | Your design | Figma Design |
| ------------- | ------------- |
| ![Screenshot 2023-06-10 214714](https://github.com/obibaadoma/potfolio-desktop/assets/102612396/593b1e07-75dd-48f8-8822-13c575edb91e) | ![Screenshot 2023-06-10 224420](https://github.com/obibaadoma/potfolio-desktop/assets/102612396/6362bc38-0755-47f8-acae-cfe72741c918) |

Reply:
done